README.md

```diff
@@ -245,6 +245,7 @@ To make sure Pancake can detect your module amongst the other hundred npm packag
 +      "plugins": [                        //only state the plugins you need here
 +        "@gov.au/pancake-sass"
 +      ],
++      "org": "@gov.au @nsw.gov.au",       //the organisations with pancake modules
```

Comment:
"the npm organisations that will be searched for pancake modules" ?

Reply:
Nice one, much better!